ghcide/src/Development/IDE/LSP/Outline.hs

```diff
@@ -69,7 +69,7 @@ documentSymbolForDecl (L (RealSrcSpan l) (TyClD _ FamDecl { tcdFam = FamilyDecl
                        t  -> " " <> t
                      )
     , _detail = Just $ pprText fdInfo
-    , _kind   = SkClass
+    , _kind   = SkInterface
```

Comment:
This one is actually for type families, right? I think the PR description is a bit misleading...

Reply:
Whoops, you're right, thanks!

Reply:
Is there any reason to leave them as `SkClass`?

Reply:
Wasn't entirely sure about type families, but in any case we should do it consistently. Thinking about it, `SkInterface` seems more appropriate than `SkClass` for type families too, since type families are somewhat analogous to type classes, but over data instead of functions.

Should I change it again?

Reply:
:hot_pepper:  suggestion: `SkFunction`. Isn't a type family *really* a type-level function? :D